heartbeat/tests/system/test_icmp.py

```diff
@@ -0,0 +1,72 @@
+import os
+import unittest
+import platform
+import socket
+import sys
+from heartbeat import BaseTest
+from elasticsearch import Elasticsearch
+from beat.beat import INTEGRATION_TESTS
+import nose.tools
+import logging
+
+
+class Test(BaseTest):
+
+    def has_admin(self):
+        if os.name == 'nt':
```

Comment:
Nice tests 👍

Reply:
Thanks... they are not working on the Travis CI.. i learned how to insert debug logs there... working on it..